apps/web/src/pages/tenants/components/UpdateTenantSidebar.tsx

```diff
@@ -82,61 +98,123 @@ export function UpdateTenantSidebar({
-            </Text>
+    <>
+      <Sidebar
+        isOpened={isOpened}
+        onClose={onClose}
+        isLoading={isLoadingTenant}
+        onSubmit={(e) => {
+          handleSubmit(onUpdateTenant)(e);
+          e.stopPropagation();
+        }}
+        customHeader={
+          <Stack h={80}>
+            <Group position="apart" spacing={120}>
+              <Controller
+                control={control}
+                name="name"
+                defaultValue=""
+                render={({ field }) => {
+                  return (
+                    <NameInput
+                      {...field}
+                      value={field.value}
+                      data-test-id="tenant-title-name"
+                      placeholder="Enter tenant name"
+                      ml={-10}
+                    />
+                  );
+                }}
+              />
+              <Tooltip label={'Delete tenant'} position="bottom">
+                <ActionIcon
+                  onClick={() => {
+                    setModalIsOpened(true);
+                  }}
+                  data-test-id="tenant-delete-btn"
+                  variant="transparent"
+                >
+                  <TrashButton>
+                    <Trash />
+                  </TrashButton>
+                </ActionIcon>
+              </Tooltip>
+            </Group>
           </Stack>
-          <Button
-            disabled={!isDirty || !isValid || isLoadingUpdate}
-            submit
-            loading={isLoadingUpdate}
-            data-test-id="update-tenant-sidebar-submit"
-          >
-            Update
+        }
+        customFooter={
+          <Group position="apart" w="100%">
+            <Stack spacing={0}>
+              <Text size={'sm'} color={colors.B40}>
+                Updated at {format(new Date(tenant.updatedAt), 'dd/MM/yyyy HH:mm')}
+              </Text>
+              <Text size={'sm'} color={colors.B40}>
+                Created at {format(new Date(tenant.createdAt), 'dd/MM/yyyy HH:mm')}
+              </Text>
+            </Stack>
+            <Button
+              disabled={!isDirty || !isValid || isLoadingUpdate}
+              submit
+              loading={isLoadingUpdate}
+              data-test-id="update-tenant-sidebar-submit"
+            >
+              Update
+            </Button>
+          </Group>
+        }
+      >
+        <TenantFormCommonFields control={control} />
+      </Sidebar>
+      <Modal
+        title={
+          <Title size={2} color={colors.warning}>
+            {' '}
+            <WarningIcon /> Delete tenant {tenant.name}?
+          </Title>
+        }
+        opened={isModalOpened}
+        onClose={() => setModalIsOpened(false)}
+        data-test-id="delete-tenant-modal"
+      >
+        <Text mb={30} size="lg" color={colors.B60}>
+          {'Deleting the tenant will stop sending notifications to all the subscribers associated with that tenant.'}
```

Comment:
```suggestion
          {'Deleting the tenant will lead to its context being unavailable during trigger event execution.'}
```
I feel this will be a better copy ;) sorry for the inconvenience